watcher/src/bin/db-dump.rs

```diff
@@ -121,14 +121,21 @@ fn display_report_status(
                 match reports.len() {
                     0 => "no reports".to_owned(),
                     1 => match &reports[0] {
-                        Some(report) => {
-                            let report_data = VerificationReportData::try_from(report)
-                                .expect("failed constructing verification report data");
-                            format!(
-                                "report available, id {} generated at {}",
-                                report_data.id, report_data.timestamp
-                            )
-                        }
+                        Some(report) => match report {
+                            EvidenceKind::Epid(report) => {
+                                let report_data = VerificationReportData::try_from(report)
+                                    .expect("failed constructing verification report data");
+                                format!(
+                                    "report available, id {} generated at {}",
+                                    report_data.id, report_data.timestamp
+                                )
+                            }
+                            EvidenceKind::Dcap(evidence) => {
+                                let signer = attestation_evidence_collector::get_block_signer_from_dcap_evidence(evidence)
+                                    .expect("failed getting signer from dcap evidence");
+                                format!("DCAP evidence available, signer {signer}")
```

Comment:
Not really sure what data we want to dump for DCAP evidence, there is no ID field or time like in the `VerificationReport`

Reply:
I think this is sufficient for now, we can update this if we ever use it again.